teamcity/PublishCSPForUnity.py

```diff
@@ -229,9 +229,12 @@ def main():
     with open(f"{args.output_directory}/package.json.meta", "w") as f:
         f.write(minimal_meta_template.render(guid=generate_guid(), importer_type="TextScript"))
 
-    # Copy readme
+    # Copy readme and create meta file
```

Comment:
It looks like there are changes here which have already been merged to staging?
For example see the change in this PR here change and [the same change](https://github.com/magnopus-opensource/connected-spaces-platform/commit/043b97ce09e05156cc355fbad0915139918b1dc7#diff-9aa94b67de50a55db1917d53ba96f25d3b73eaad93346261e2f0c0a2a592f85f) from a merge that has already gone into staging